packages/knowledge/src/chartID.ts

```diff
@@ -77,7 +77,6 @@ export const CHART_ID_OPTIONS = [
   'bubble_map',
   'bubble_light_map',
   'packed_circles',
-  'polar_treemap',
```

Comment:
did u run tests or lerna boot? this could make one test case fail

Reply:
I didn't. 
I've solved it now.